spec/services/catalog/controllers/admin-catalog-item-controller-spec.js

```diff
@@ -100,13 +96,13 @@ describe('Catalog Service Controller', function () {
 			identity: {
 				platform: {id: 'apple-ios'},
 				user: {id: 'user-id'},
-				audience: 'admin'
+				audience: ['admin']
```

Comment:
Hmmm..... Those collapsed 'outdated' things up there have valid comments in them.